gdc_client/download/client.py

```diff
@@ -320,24 +326,3 @@ def parallel_download(self, stream):
                     file_id, e))
                 if self.debug:
                     raise
-
-
-class GDCUDTDownloadClient(GDCHTTPDownloadClient):
```

Comment:
This is probably ok. I don't think there are plans to use UDT in the near future.

Reply:
Yeah, just remove it :(